controller/helper/process-survey-instances.js

```diff
@@ -0,0 +1,60 @@
+'use strict';
+
+/**
+ * @module controller/helper/process-survey-instances
+ */
+
+/**
+ * Takes in a Survey Instances and processes it to get complence chart details
+ * @param {Surveys} list of survey instances
+ * @returns {Object}  Complience chart data
+ */
+function processSurveyInstances (surveys) {
+    return {
+      labels:pickupSubmissionTime(surveys),
+      datasets:calculateTimeLeft(surveys)
+    };
+}
+
+function pickupSubmissionTime(surveys) {
+  var dates = [];
+  dates.length = surveys.length;
+  for (var i = 0; i < surveys.length; i++) {
+    dates[i] = formatDate(surveys[i].startTime);
+  }
+  return dates;
```

Comment:
for reformatting arrays use the `map` method.

reference: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map